server/controllers/connection-status.js

```diff
@@ -0,0 +1,44 @@
+const {
+    queryDynamoByChannel,
+    updateConnectionStatus,
+} = require('../lib/dynamoDao');
+
+// TO BE USED LATER AS SOME KIND OF TYPESAFE ENUM VALIDATION
+const CONNECTION_STATUS = {
```

Comment:
We can try to throw in a typescript enum in here if you want. I'm not partial, though the last time I tried to use enums in Typescript, it ended poorly.